Localization/Resources/es-ES/winget.resw

```diff
@@ -1268,4 +1275,78 @@ Especifique uno de ellos con la opción '--source' para continuar.</value>
+  </data>
+  <data name="PurgeArgumentDescription" xml:space="preserve">
+    <value>Elimina todos los archivos y directorios del directorio del paquete (archivo portátil)</value>
+  </data>
+  <data name="RenameArgumentDescription" xml:space="preserve">
+    <value>Valor para cambiar el nombre del archivo ejecutable (archivo portátil)</value>
+  </data>
+  <data name="WaitArgumentDescription" xml:space="preserve">
+    <value>Pide al usuario que presione cualquier tecla antes de salir</value>
+  </data>
+  <data name="InstallerElevationExpected" xml:space="preserve">
+    <value>El instalador solicitará que se ejecute como administrador; se espera un mensaje.</value>
+  </data>
+  <data name="InstallerProhibitsElevation" xml:space="preserve">
+    <value>El instalador no se puede ejecutar desde un contexto de administrador.</value>
+  </data>
+  <data name="ModifiedPathRequiresShellRestart" xml:space="preserve">
+    <value>Variable de entorno PATH modificada; reinicie el shell para usar el nuevo valor.</value>
+  </data>
+  <data name="ProductCodeArgumentDescription" xml:space="preserve">
+    <value>Filtros con el código de producto</value>
+  </data>
+  <data name="PortableRegistryCollisionOverridden" xml:space="preserve">
+    <value>Ya existe un paquete portátil con el mismo nombre pero de otro origen; continuar debido a --force</value>
+  </data>
+  <data name="ReparsePointsNotSupportedError" xml:space="preserve">
+    <value>El volumen no admite puntos de análisis</value>
+  </data>
+  <data name="ReservedFilenameError" xml:space="preserve">
+    <value>El nombre de archivo especificado no es un nombre de archivo válido</value>
+  </data>
+  <data name="OverwritingExistingFileAtMessage" xml:space="preserve">
+    <value>Sobrescribiendo archivo existente:</value>
+  </data>
+  <data name="NoPackageSelectionArgumentProvided" xml:space="preserve">
+    <value>No se ha proporcionado ningún argumento de selección de paquete; vea la ayuda para obtener más información sobre cómo buscar un paquete.</value>
+  </data>
+  <data name="PortableInstallFailed" xml:space="preserve">
+    <value>Error en la instalación del archivo portátil; Limpiando...</value>
+  </data>
+  <data name="BothPurgeAndPreserveFlagsProvided" xml:space="preserve">
+    <value>Se proporcionan los argumentos "purge" y "preserve"</value>
+  </data>
+  <data name="PortableHashMismatchOverridden" xml:space="preserve">
+    <value>Se ha modificado el archivo EXE portátil; continuar debido a --force</value>
+    <comment>{Locked="--force"}</comment>
+  </data>
+  <data name="PortableHashMismatchOverrideRequired" xml:space="preserve">
+    <value>No se puede quitar el archivo EXE portátil porque se ha modificado; para invalidar esta comprobación, use --force</value>
+    <comment>{Locked="--force"}</comment>
+  </data>
+  <data name="FilesRemainInInstallDirectory" xml:space="preserve">
+    <value>Los archivos permanecen en el directorio de instalación:</value>
```

Comment:
This doesn't sound right to me how I imagine it being used. It says "The files remain [...]:". I'd think it should be something like "Estos archivos permanecen [...]:" ("these files remain [...]") or "Los siguientes archivos permanecen [...]:" ("the following files remain [...]:")